Build Fails With Pipewire: "was not declared in this scope"

<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
<!--- Read the following link before you create a new problem report: https://kodi.wiki/view/HOW-TO:Submit_a_bug_report  -->
## Bug report
### Describe the bug
Here is a clear and concise description of what the problem is:
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
<!--- A bug report that is not clear will be closed -->
<!--- Put your text below this line -->
```
<pathTo>/xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp: In member function 'virtual bool AE::SINK::CAESinkPipewire::Initialize(AEAudioFormat&, std::string&)':
<pathTo>/xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp:316:26: error: 'PW_KEY_MEDIA_TYPE' was not declared in this scope
  316 |       SPA_DICT_ITEM_INIT(PW_KEY_MEDIA_TYPE, "Audio"),
      |                          ^~~~~~~~~~~~~~~~~
/usr/include/spa-0.2/spa/utils/dict.h:41:64: note: in definition of macro 'SPA_DICT_ITEM_INIT'
   41 | #define SPA_DICT_ITEM_INIT(key,value) (struct spa_dict_item) { key, value }
      |
```


## Expected Behavior
Here is a clear and concise description of what was expected to happen:
<!--- Tell us what should happen -->
<!--- Put your text below this line -->
Kodi should build just fine with pipewire already installed.


## Actual Behavior
<!--- Tell us what happens instead -->
<!--- Put your text below this line -->
The error above (along with related "not declared in this scope" issues)


## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->
<!--- Put your text below this line -->



### To Reproduce
Steps to reproduce the behavior:
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
<!--- Put your text below this line -->
1.  `cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DENABLE_INTERNAL_FFMPEG=OFF -DENABLE_INTERNAL_CROSSGUID=OFF -DENABLE_DVDCSS=OFF -DENABLE_OPTICAL=OFF -DENABLE_AIRTUNES=OFF -DAPP_RENDER_SYSTEM=gl .. 2>&1 | tee cmake.log`
2. `PREFIX=/usr/local make 2>&1 | tee make.log`



### Debuglog
<!--- Put your text below this line -->
<!--- A debuglog is always mandatory when creating an issue. Provide one! -->
The debuglog can be found here:
[cmake.log](https://github.com/xbmc/xbmc/files/6598526/cmake.log)
[make.log](https://github.com/xbmc/xbmc/files/6598531/make.log)



### Screenshots 
Here are some links or screenshots to help explain the problem:
<!--- Put your text below this line -->



## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:
<!--- How has this bug affected you? What were you trying to accomplish? -->
<!--- Put your text below this line -->



### Your Environment
Used Operating system:
<!--- Include as many relevant details about the environment you experienced the bug in -->
<!--- Put your text below this line. Checkboxes can easily be ticked once issue is created -->
 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [X] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Archlinux
 - Kodi version: Latest git master fetched commit fc8ff86395a9e804105cdfed7f268a923f074df3



<!--- End of this issue -->
*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

--- a/xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp
+++ b/xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp
@@ -17,6 +17,7 @@
 #include "cores/AudioEngine/Sinks/pipewire/PipewireThreadLoop.h"
 #include "utils/log.h"
 
+#include <pipewire/keys.h>
 #include <spa/param/audio/raw.h>
 
 namespace

